joern-cli/frontends/c2cpg/src/test/scala/io/joern/c2cpg/querying/NodeTypeStarterQueryTests.scala

```diff
@@ -117,8 +117,8 @@ class NodeTypeStarterQueryTests extends C2CpgSuite {
     val method1 = cpg.method.name("main").head
     val method2 = cpg.method.name("libfunc").head
 
-    cpg.id(method1.id).l shouldBe Seq(method1)
-    cpg.ids(method1.id, method2.id).l shouldBe Seq(method1, method2)
+    cpg.all.id(method1.id).l shouldBe Seq(method1)
+    cpg.all.id(method1.id, method2.id).l shouldBe Seq(method1, method2)
```

Comment:
This is very bad, because it is "search in iterator". We actually have O(1) access to nodes via id, we just need to offer an API for it.